app/models/patient.rb

```diff
@@ -65,6 +69,7 @@ class Patient
             :primary_phone,
             :initial_call_date,
             :created_by,
+            :line,
```

Comment:
Does the addition of `:line` to the `validates` list make it a required field? If so, what happens to all the existing Patients in production without one?